API.md

```diff
@@ -78,28 +78,28 @@ log.append({ some: 'data' })
   .then(log => log.append('text'))
   .then(log => console.log(log.values))
 
-// [ 
-//   { 
+// [
+//   {
 //     hash: 'QmV1KFxZnaguPFp57PMXz5Dd1tf6z9U8csJQoy4xWDzzts',
 //     id: 'A',
 //     payload: { some: 'data' },
 //     next: [],
 //     v: 0,
-//     clock: LamportClock { id: 'A', time: 0 } 
+//     clock: LamportClock { id: 'A', time: 0 }
 //   },
 //   { hash: 'QmSxe4Shd7jt4ExyoBjtvgi1UabNKrZfRJKptwUmSa843u',
 //     id: 'A',
 //     payload: 'text',
 //     next: [ 'QmV1KFxZnaguPFp57PMXz5Dd1tf6z9U8csJQoy4xWDzzts' ],
 //     v: 0,
```

Comment:
Additionally, we should probably increment this from 0 to 1 as well. See the thread and comment here: https://github.com/orbitdb/ipfs-log/issues/211#issuecomment-450730115

cc @haadcode and @shamb0t

Reply:
Yep, will change.

Reply:
Thank you!